lib/version_utils.pm

```diff
@@ -331,6 +331,7 @@ sub is_tumbleweed {
     return 0 unless check_var('DISTRI', 'opensuse');
     return 1 if get_var('VERSION') =~ /Tumbleweed|Slowroll/;
     return 1 if is_gnome_next;
+    return 1 if get_var('AGAMA_PRODUCT') =~ /Tumbleweed/;
```

Comment:
I don't know why this is needed...

Reply:
we need `opensuse_welcome_applicable` for [**opensuse-welcome**](https://openqa.opensuse.org/tests/4544704#step/first_boot/12)

```perl
   my @tags = qw(generic-desktop emergency-shell emergency-mode);
    push(@tags, 'opensuse-welcome') if opensuse_welcome_applicable;
```
in `opensuse_welcome_applicable` 

```perl
return 0 unless is_tumbleweed || is_leap(">=15.3");
```

Reply:
this variable will disappear when our project with puppeteer will be more consolidated, in general avoid using any openQA variables. Can we just switch VERSION to Tumbleweed, which is really what we install?

Reply:
OK, thanks for the information.